proto/osmosis/epochs/genesis.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+package osmosis.epochs.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/c-osmosis/osmosis/x/epochs/types";
+
+message EpochInfo {
+    string identifier = 1;
+    google.protobuf.Timestamp start_time = 2 [
+        (gogoproto.stdtime) = true,
+        (gogoproto.nullable) = false,
+        (gogoproto.moretags) = "yaml:\"start_time\""
+    ];
+    google.protobuf.Duration duration = 3 [
+        (gogoproto.nullable) = false,
+        (gogoproto.stdduration) = true,
+        (gogoproto.jsontag) = "duration,omitempty",
+        (gogoproto.moretags) = "yaml:\"duration\""
+    ];
+    int64 current_epoch = 4;
+    google.protobuf.Timestamp current_epoch_start_time = 5 [
+        (gogoproto.stdtime) = true,
+        (gogoproto.nullable) = false,
+        (gogoproto.moretags) = "yaml:\"current_epoch_start_time\""
+    ];
+    bool epoch_counting_started = 6;
+    bool current_epoch_ended = 7;
+}
```

Comment:
Proposal to add a new EpochInfo is not required?

Reply:
What do you mean by this?

Reply:
I don't understand what you mean here

Reply:
I think it won't be required to add more epochs by governance now @ValarDragon @ahmedaly113

Reply:
Agreed. Currently we have polo rewards set for once a week, not a variable timer set by the pool, which seems sufficient for launch imo